Microsoft.TeamServices.Samples.Client/Release/ReleasesSample.cs

```diff
@@ -0,0 +1,512 @@
+
+            // Get a release client instance
+            VssConnection connection = Context.Connection;
+            ReleaseHttpClient2 releaseClient = connection.GetClient<ReleaseHttpClient2>();
+            List<ReleaseDefinition> releaseDefinitions = releaseClient.GetReleaseDefinitionsAsync(project: projectName).Result;
+            int releaseDefinitionId = releaseDefinitions.FirstOrDefault().Id;
+            List<Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Release> releases = releaseClient.GetReleasesAsync(project: projectName, definitionId: releaseDefinitionId).Result;
+
+            // Show the approvals
+            foreach (Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Release release in releases)
+            {
+                Console.WriteLine("{0} {1}", release.Id.ToString().PadLeft(6), release.Status);
+            }
+
+            return releases;
+        }
+
+        [ClientSampleMethod]
+
+        public Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Release GetARelease()
+        {
+            string projectName = ClientSampleHelpers.FindAnyProject(this.Context).Name;
+
+            // Get a release client instance
+            VssConnection connection = Context.Connection;
+            ReleaseHttpClient releaseClient = connection.GetClient<ReleaseHttpClient>();
+
+            List<Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Release> releases = releaseClient.GetReleasesAsync(project: projectName).Result;
+
+            int releaseId = releases.FirstOrDefault().Id;
+
+            // Show a releaes
+            Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Release release = releaseClient.GetReleaseAsync(project: projectName, releaseId: releaseId).Result;
+
+            Console.WriteLine("{0} {1}", release.Id.ToString().PadLeft(6), release.Name);
+
+            return release;
+        }
+
+        [ClientSampleMethod]
+
+        public Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Release CreateARelease()
+        {
+            string projectName = ClientSampleHelpers.FindAnyProject(this.Context).Name;
+
+            // Get a release client instance
+            VssConnection connection = Context.Connection;
+            ReleaseHttpClient releaseClient = connection.GetClient<ReleaseHttpClient>();
+
+            List<ReleaseDefinition> releaseDefinitions = releaseClient.GetReleaseDefinitionsAsync(project: projectName).Result;
+            int releaseDefinitionId = releaseDefinitions.FirstOrDefault().Id;
+
+            Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Release release = CreateRelease(releaseClient, releaseDefinitionId, projectName);
+
+            Console.WriteLine("{0} {1}", release.Id.ToString().PadLeft(6), release.Name);
+
+            return release;
+        }
+
+        [ClientSampleMethod]
+
+        public Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Release UpdateReleaseEnvironment()
```

Comment:
"Update" is a fairly nebulous term and can mean a lot of different things. In the case of this sample method, I would give it a real clear name like `StartDeployment` (or whatever makes the most sense for what it's showing). Most users aren't going to think to look for this type of operation under a "Update" sample.